docs/src/build_tips.md

```diff
@@ -127,6 +127,34 @@ Example of packages using Rust:
 
 	The Rust toolchain currently used does not work with the `i686-w64-mingw32` (32-bit Windows) platform.
 
+ ## C builds
+
+ If your library has no build system like Make, CMake, Meson, or Autoconf, you may need to use the C compiler directly.  The C compiler is stored in the `CC` environment variable, and you can direct output to `libdir` (shared libraries) and `bindir` (executables).
+
```

Comment:
I don't want to give the impression that this are the exact commands one shall run, it's rather to give an idea (I can't say how many times people take these instructions almost verbatim rather than as indication)
```suggestion
As a high-level example:

```